crates/matrix-sdk-base/src/client.rs

```diff
@@ -1070,7 +1071,16 @@ impl BaseClient {
 
     pub(crate) fn apply_changes(&self, changes: &StateChanges, trigger_room_list_update: bool) {
         if changes.account_data.contains_key(&GlobalAccountDataEventType::IgnoredUserList) {
-            self.ignore_user_list_changes.set(());
+            if let Some(event) =
+                changes.account_data.get(&GlobalAccountDataEventType::IgnoredUserList)
+            {
+                if let Ok(event) = event.deserialize_as::<IgnoredUserListEvent>() {
```

Comment:
Do we want to log something in the error case here?

Reply:
Yep, sorry, I should've done that